src/user.rs

```diff
@@ -182,28 +145,31 @@ impl User {
             ) {
                 Ok(result)
             } else {
-                Err(JsonSaveError::WriteFile)
+                Err(Error::WriteFile)
             }
         } else {
-            Err(JsonSaveError::Serialize)
+            Err(Error::Serialize)
         }
     }
 
-    pub async fn load(id: &ID) -> Result<Self, JsonLoadError> {
-        if let Ok(json) =
-            tokio::fs::read_to_string(USER_FOLDER.to_owned() + &id.to_string() + ".json").await
-        {
+    pub async fn load(id: &ID) -> Result<Self> {
+        let filename = USER_FOLDER.to_owned() + &id.to_string() + ".json";
```

Comment:
I personally don't do concatenation of strings in rust because I always use `format` which allows me to not clone the types and use references instead.